goon-universe/queries.js

```diff
@@ -863,6 +892,161 @@ function loadScenarioCSV(scenario_csv_string){
     
 }
 
+async function getMCQResponse(pageOrder,submissionID, questionID){
+    const thisQuery='select response.*, mcq_response.* from response, mcq_response, pages where pages.order=$1 AND response.page_id=pages.id AND response.submission_id=$2 AND response.id=mcq_response.id AND mcq_response.question_id=$3'
+    const client = await pool.connect();
+    try {
+        const queryReturn= await client.query(thisQuery, [pageOrder, submissionID, questionID]);
+        let mcqResponse=queryReturn.rows[0];
+        return mcqResponse;
+    } catch (e) {
+        throw e;
+
+    } finally {
+        client.release();
+    }
+}
+function getInitActionResponse(submissionID, questionID, callback){
+    getMCQResponse(INIT_ACTION,submissionID, questionID).then((result) => callback(result));
+}
+
+function getFinalActionResponse(submissionID, questionID, callback){
+    getMCQResponse(FINAL_ACTION,submissionID, questionID).then((result) => callback(result));
+}
```

Comment:
These functions are being called in app.js as `getInitActionResponse(studentID, scenarioID, function(result)`

Are there query functions that app.js can use to get the submissionID and questionID? If not, this should be re-written to take in a studentID and scenarioID.